### PR TITLE
Update important-upgrade-notes.rst

### PR DESCRIPTION
**Pull request by amyblais**
Related to https://community.mattermost.com/core/pl/tpad8fpqwbnhjrmykndkm5ogdr.

### CODE REVIEW

**Comment by JtheBAB** [comment]
> Activating "Skip Certificate verification" fixes the issue.

This is only a hotfix. The real fix is to create a new certificate on the AD/LDAP Server with the SAN inside :)

**Review by sbishel — APPROVED** [review]

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -24,6 +24,10 @@ Important Upgrade Notes
...
 |                                                    | and ``GetPostsAroundLastUnread`` now require an additional collapsedThreads parameter to be passed. Any client making use of these functions will need to update |
 |                                                    | them when upgrading its dependencies.                                                                                                                            |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | `A breaking change was introduced when upgrading the Go version to v1.15.5 <https://golang.org/doc/go1.15#commonname>`_ where user logins fail with AD/LDAP Sync |
+|                                                    | when the certificate of the LDAP server has no Subject Alternative Name (SAN) in it. Creating a new certificate on the AD/LDAP Server with the SAN inside fixes  |
```
Comment (cwarnermm) [inline source/administration/important-upgrade-notes.rst:29]:
```suggestion
|                                                    | When the certificate of the LDAP Server has no Subject Alternative Name (SAN) in it. Creating a new certificate on the AD/LDAP Server with the SAN inside fixes this.  |
```

File: source/administration/changelog.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ Mattermost v5.32.0 contains low level security fixes. [Upgrading](https://docs.m
 ### Breaking Changes
  - ``ExperimentalChannelOrganization``, ``EnableXToLeaveChannelsFromLHS``, ``CloseUnusedDirectMessages``, and ``ExperimentalHideTownSquareinLHS`` settings are only functional if the Legacy Sidebar (``EnableLegacySidebar``) is enabled since they are not compatible with the new sidebar experience. ``ExperimentalChannelSidebarOrganization`` has been deprecated, since the [new sidebar is now enabled for all users](https://mattermost.com/blog/custom-collapsible-channel-categories/). 
  - Breaking changes to the Golang client API were introduced: ``GetPostThread``, ``GetPostsForChannel``, ``GetPostsSince``, ``GetPostsAfter``, ``GetPostsBefore``, and ``GetPostsAroundLastUnread`` now require an additional collapsedThreads parameter to be passed. Any client making use of these functions will need to update them when upgrading its dependencies.
+ - [A breaking change was introduced when upgrading the Go version to v1.15.5](https://golang.org/doc/go1.15#commonname) where user logins fail with AD/LDAP Sync when the certificate of the LDAP server has no Subject Alternative Name (SAN) in it. Creating a new certificate on the AD/LDAP Server with the SAN inside fixes this.
```
Comment (cwarnermm) [inline source/administration/changelog.md:20]:
```suggestion
 - [A breaking change was introduced when upgrading the Go version to v1.15.5](https://golang.org/doc/go1.15#commonname) where user logins fail with AD/LDAP Sync when the certificate of the LDAP Server has no Subject Alternative Name (SAN) in it. Creating a new certificate on the AD/LDAP Server with the SAN inside fixes this.
```

**Review by cwarnermm — APPROVED** [review]
Non-blocking suggestions provided inline.

**Pull request merged**